juno/assetlist.json

```diff
@@ -552,6 +552,50 @@
         "logo_URIs": {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/hopers.png"
         }
+      },
+      {
+        "description": "RED",
+        "type_asset": "cw20",
+        "address": "juno1g647t78y2ulqlm3lss8rs3d0spzd0teuwhdvnqn92tr79yltk9dq2h24za",
+        "denom_units": [
+          {
+            "denom": "cw20:juno1g647t78y2ulqlm3lss8rs3d0spzd0teuwhdvnqn92tr79yltk9dq2h24za",
+            "exponent": 0
+          },
+          {
+            "denom": "red",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:juno1g647t78y2ulqlm3lss8rs3d0spzd0teuwhdvnqn92tr79yltk9dq2h24za",
+        "name": "Red",
+        "display": "Red",
+        "symbol": "RED",
+        "logo_URIs": {
+          "png": "https://user-images.githubusercontent.com/521978/212457442-072250b6-501b-44d4-9fb2-291e69d2fa53.jpg"
+        }
+      },
+      {
+        "description": "BLUE",
+        "type_asset": "cw20",
+        "address": "juno14q8kk464fafql2fwmlsgvgcdl6h2csqpzv4hr025fmcvgjahpess32k0j7",
+        "denom_units": [
+          {
+            "denom": "cw20:juno14q8kk464fafql2fwmlsgvgcdl6h2csqpzv4hr025fmcvgjahpess32k0j7",
+            "exponent": 0
+          },
+          {
+            "denom": "blue",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:juno14q8kk464fafql2fwmlsgvgcdl6h2csqpzv4hr025fmcvgjahpess32k0j7",
+        "name": "Blue",
+        "display": "Blue",
```

Comment:
display should be 'blue' (lowercase 'b'), as it must match one of the denom_units